v0/scripts/PublishPrices.js

```diff
@@ -74,7 +78,7 @@ async function getBarchartPrice(asset) {
 
 // Gets the Coinbase price for an asset or throws.
 async function getCoinbasePrice(asset) {
-  const url = `https://api.coinbase.com/v2/prices/${asset}/spot`;
+  const url = `https://api.coinbase.com/v2/prices/${asset}-USD/spot`;
```

Comment:
Is this change incompatible with PR #430, not sure which approach we prefer.

Reply:
I think I prefer #430 since it makes the config more explicit.